theano/misc/elemwise_openmp_speedup.py

```diff
@@ -23,7 +23,7 @@ def runScript(N):
     if err:
         print(err)
         sys.exit()
-    return list(map(float, out.split(" ")))
+    return list(map(float, out.decode().split(" ")))
```

Comment:
thanks for looking into that. In the file theano/sandbox/cuda/nvcc_compiler.py, we do something like:

from locale import getpreferredencoding
from theano.compat import decode, decode_with
nvcc_stdout_raw, nvcc_stderr_raw = p.communicate()[:2]
console_encoding = getpreferredencoding()
nvcc_stdout = decode_with(nvcc_stdout_raw, console_encoding)
nvcc_stderr = decode_with(nvcc_stderr_raw, console_encoding)

This also allow to work with different language and we know it work with python 2.6 that we still support (but not for long). Can you update your PR to follow this more tested pattern?

thanks